README.md

```diff
@@ -1,4 +1,4 @@
-# IBM Cloud <module_name> - Terraform Module
+# IBM Cloud VPC VSI - Terraform Module
```

Comment:
Update test.yml file with code to write test results into slack channel (Take code from https://github.com/terraform-ibm-modules/terraform-ibm-module-template repo)

Reply:
done